### ISSUE TITLE
Add state.sls_id for salt-ssh

### ISSUE THREAD

**Issue by rmarchei** [saltstack/salt#44733]
### Description of Issue/Question

[sls_id](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.state.html#salt.modules.state.sls_id) module doesn't work with `salt-ssh`

### Steps to Reproduce Issue
Running `state.sls_id` with `salt-ssh` fails:
```
# salt-ssh remote_host state.sls_id my_state my_module
remote_host:
    Data failed to compile:
----------
    No matching sls found for 'my_module' in env 'base'
```

### Versions Report
```
# salt-call -V
Salt Version:
           Salt: 2017.7.2

Dependency Versions:
           cffi: 1.5.2
       cherrypy: 3.5.0
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: 0.24.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.3.7
      pycparser: 2.14
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.24.0
         Python: 2.7.12 (default, Nov 20 2017, 18:23:56)
   python-gnupg: 0.3.8
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: 0.2.4
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-97-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by gtmanfred** [comment]
This is caused by the fact that the state module for salt-ssh does not have an sls_id written.

https://github.com/saltstack/salt/blob/2017.7/salt/client/ssh/wrapper/state.py

For salt-ssh, certain states have to be wrapped in order for them to work with communication with the salt master. 

I am going to mark this as a feature request to add `sls_id` to the salt-ssh state wrapper.

Thanks,
Daniel

**Comment by rmarchei** [comment]
Apparently it has just been added in https://github.com/saltstack/salt/pull/44659 and it will be available in 2017.7.3! Closing this one

**Comment by gtmanfred** [comment]
ahh, perfect!
